src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -213,7 +213,7 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
 
             var artifactMetadata = new ArtifactMetadata
             {
-                Alias = "Roslyn-Signed", // This is the alias in the Release definition.
+                Alias = "InputBuild", // This is the alias in the Release definition.
```

Comment:
Do other build definitions need to specify an alias?  What's the benefit?

Reply:
@brettfo We use the same release queue for all definitions. When it was setup the alias was called "Roslyn-Signed" because the default alias name is just the name of the build definition it's connected to. I renamed it to InputBuild so it's clearer it's just any random build.